src/controller/SetUpCodePairer.cpp

```diff
@@ -405,9 +406,17 @@ void SetUpCodePairer::NotifyCommissionableDeviceDiscovered(const Dnssd::CommonRe
     ConnectToDiscoveredDevice();
 }
 
-void SetUpCodePairer::CommissionerShuttingDown()
+bool SetUpCodePairer::StopPairing(NodeId remoteId)
 {
+    if ((remoteId != kUndefinedNodeId && remoteId != mRemoteId) || mRemoteId == kUndefinedNodeId)
+    {
+        return false; // Not discovering the specified node (or any node)
+    }
```

Comment:
Maybe:
```
VerifyOrReturnValue(mRemoteId != kUndefinedNodeId, false);
VerifyOrReturnValue(remoteId == kUndefinedNodeId || remoteId == mRemoteId, false);
```

It is not very clear from the code that `kUndefinedNodeId` is a joker for dismissing the current target. Can you add it to the documentation of `StopPairing`